TestCases/compliance-level-3/0080-feel-getvalue-function/0080-feel-getvalue-function-test-01.xml

```diff
@@ -100,14 +100,22 @@
     </testCase>
 
     <testCase id="decision_011">
-        <description>null value stored in context</description>
-        <resultNode name="decision_011" type="decision">
+        <description>null map and key params</description>
+        <resultNode errorResult="true" name="decision_011" type="decision">
```

Comment:
Test case:
https://github.com/dmn-tck/tck/blob/9f8ba50870259c971b5980d031fe73224635be09/TestCases/compliance-level-3/0080-feel-getvalue-function/0080-feel-getvalue-function.dmn#L113

So this is about extracting the entry `null` in a `null` context, not the original submission.
Reflecting on the correct test DMN model decision logic.
Additionally, this is an error case, putting the flag.